.gen/pipeline/version.go

```diff
@@ -1,3 +1,3 @@
 package pipeline
 
-const PipelineVersion = "0.60.0"
+const PipelineVersion = "pr-3260"
```

Comment:
**Note**: don't forget to update the Pipeline version in the Makefile and regenerate the client files when the Pipeline API release is available. (Only commenting because I found no corresponding TODOs in the PR description.)

Reply:
fixed